components/suggestion/emoticon_provider.tsx

```diff
@@ -148,7 +164,7 @@ export default class EmoticonProvider extends Provider {
             }
         }
 
-        const sortEmojisHelper = (a, b) => {
+        const sortEmojisHelper = (a: any, b: any) => {
```

Comment:
Also there is a bug here we should not have passed the a and b directly to the compare function since it's not of type Emoji